### PR TITLE
add compiler path to debug configs details

### PR DESCRIPTION
**Pull request by elahehrashedi**
Add more details to debug configs:

image

### CODE REVIEW

**Comment by Colengms** [comment]
Is it possible to use an additional line?  If not, perhaps we could omit the "compiler:" portion, so more of the path is visible?

**Review by bobbrow — APPROVED** [review]

File: Extension/src/Debugger/configurationProvider.ts [hunk 1]
```diff
@@ -37,11 +37,29 @@ export enum TaskConfigStatus {
...
+                item.detail = localize("detected.task", "Detected Task");
+                break;
+            }
+            default : {
+                break;
+            }
+        }
+        if (item.configuration.taskDetail) {
+            // Add the compiler path of the preLaunchTask to the description of the debug configuration.
+            item.detail = (item.detail ? item.detail : "") + " (" + item.configuration.taskDetail + ")";
```
Comment (sean-mcmanus) [inline Extension/src/Debugger/configurationProvider.ts]: Can this be simplified to `(item.detail ?? "")`?

**Comment by bobbrow** [comment]
> Is it possible to use an additional line? If not, perhaps we could omit the "compiler:" portion, so more of the path is visible?

I don't believe it is possible to have a 3rd line.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**